crates/bevy_render/src/camera/camera.rs

```diff
@@ -83,6 +83,71 @@ pub struct ComputedCameraValues {
     old_viewport_size: Option<UVec2>,
 }
 
+/// How much energy a `Camera3d` absorbs from incoming light.
+///
+/// <https://en.wikipedia.org/wiki/Exposure_(photography)>
+#[derive(Component)]
+pub struct ExposureSettings {
+    /// <https://en.wikipedia.org/wiki/Exposure_value#Tabulated_exposure_values>
+    pub ev100: f32,
+}
+
+impl ExposureSettings {
+    pub const EV100_SUNLIGHT: f32 = 15.0;
+    pub const EV100_OVERCAST: f32 = 12.0;
+    pub const EV100_INDOOR: f32 = 7.0;
+
+    pub fn from_physical_camera(physical_camera_parameters: PhysicalCameraParameters) -> Self {
+        Self {
+            ev100: physical_camera_parameters.ev100(),
+        }
+    }
+
+    /// Converts EV100 values to exposure values.
+    #[inline]
+    pub fn exposure(&self) -> f32 {
+        1.0 / (2.0f32.powf(self.ev100) * 1.2)
```

Comment:
where does the 1.2 come from?

Reply:
i ctrl+f'd 1.2 in the wiki article for exposure and didnt find anything

Reply:
Not sure either :thinking:

Reply:
looks like its present from the very first commit. https://github.com/bevyengine/bevy/pull/8407/commits/432c427f3249e0d41ed7f3a5749d09a4e7788e99#diff-79ddab7b083d01b7da117dcbd27b3e5effe647583ac8c6345ee2255db851d667R105

@superdump

Reply:
furthermore, ev100 returns log2 of a thing and then exposure immediately exp2's it back, and doesn't even take advantage of being able to negate it before exp2'ing to skip the rcp. seeing as the only place ev100 is used is in exposure, maybe its better to
```suggestion
        1.2 * self.shutter_speed_s * self.sensitivity_iso / (self.aperture_f_stops * self.aperture_f_stops * 100.0)
```
and add tests to ensure ev100 agrees with exposure

Reply:
scratch that, i see its not a method call here anymore. This would still work though (assuming the 1.2 is desired)
```suggestion
        2.0f32.powf(-self.ev100) * 1.2
```

Reply:
It is explained in section 8.1.3 of the Filament shading description: https://google.github.io/filament/Filament.md.html